From f912ead404ffa24db7f4aee527aff411db39262a Mon Sep 17 00:00:00 2001
From: Gabriel Craciunescu <crazy@frugalware.org>
Date: Wed, 11 Apr 2018 17:17:06 +0200
Subject: [PATCH] Revert "x86/ACPI/cstate: Allow ACPI C1 FFH MWAIT use on AMD systems"

This reverts commit 5209654a46ee71137ad9b06da99d4ef2794475af.
Please see: https://community.amd.com/thread/224000

---
 arch/x86/kernel/acpi/cstate.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/x86/kernel/acpi/cstate.c b/arch/x86/kernel/acpi/cstate.c
index dde437f5d14f..8233a630280f 100644
--- a/arch/x86/kernel/acpi/cstate.c
+++ b/arch/x86/kernel/acpi/cstate.c
@@ -167,8 +167,7 @@ static int __init ffh_cstate_init(void)
 {
 	struct cpuinfo_x86 *c = &boot_cpu_data;
 
-	if (c->x86_vendor != X86_VENDOR_INTEL &&
-	    c->x86_vendor != X86_VENDOR_AMD)
+	if (c->x86_vendor != X86_VENDOR_INTEL)
 		return -1;
 
 	cpu_cstate_entry = alloc_percpu(struct cstate_entry);
-- 
2.17.0

